internal/tui/suggest/ai_message.go

```diff
@@ -114,10 +140,21 @@ func (m *AIMessageModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		switch msg.String() {
 		case "ctrl+c", "q":
 			return m, tea.Quit
-		case "e":
```

Comment:
edit and regenerate would be useful commands so we shall not forget to implement them